### ISSUE TITLE
Test configuration

### ISSUE THREAD

**Issue by kevee** [quailjs/quail#79]
Certain tests need configuration based on the guideline being used. For example, a color contrast test might fail on different contrast ratios.

An example YAML configuration would be:

```
colorContrast
  guidelines
    wcag:
      configuration:
        ratio: 4.5
      1.3.1:
        techniques:
          - "X1"
```

This would, however, require a new quail-wide configuration option to label what guideline is to be used.
